### ISSUE TITLE
Stranglethorn vale , The fishing contest.  

### ISSUE THREAD

**Issue by Dragoonduneman** [slowtorta/turtlewow-bug-tracker#5080]
Ok the npc Riggle Bassbait.   Is behaving as if he's in a completed state.   He did not yell for when the Fishing contest started only when it ended.   I have reason to believe that he's stuck in a completed state and never unlocking each week.  

NPC: https://database.turtle-wow.org/?npc=15235

Quest: https://database.turtle-wow.org/?quest=Master Angler

**Expected behavior.  Ok every week he's suppose to announced when the contest has started.   Once the winner is found, He suppose to yell out announcing to the zone that there has been a winner.

**Comment by twowTurtReynolds** [comment]
@Dragoonduneman Has this occured in the weeks since? Or was this a one time bug?

**Comment by Dragoonduneman** [comment]
hi , i only observed it once time , since it was the only time i did the
event.   I havent revisit it to see if it was recurring.   But it was
indeed strange that no winner was announced and that the gobbo girl that
next to him open the vendor at the start ... when it should only be open
after halfway or a winner was won.   According to wowhead ... And that
There was no quest to accept or pop up.   I suppose , we can revisit weekly
to see if any winner was won or not.

**Comment by twowTurtReynolds** [comment]
Indeed, if you can revisit this event and fill out the bug report with more info, that would help a lot.
